Tests/StreamChatTests/Workers/Background/ConnectionRecoveryHandler_Tests.swift

```diff
@@ -504,6 +505,7 @@ final class ConnectionRecoveryHandler_Tests: XCTestCase {
 
         XCTAssertCall(RetryStrategy_Spy.Signature.resetConsecutiveFailures, on: mockRetryStrategy, times: 1)
         XCTAssertCall("syncLocalState(completion:)", on: mockChatClient.mockSyncRepository, times: 1)
+        XCTAssertTrue(mockChatClient.mockExtensionLifecycle.receivedIsReceivingEvents == true)
```

Comment:
nit: Consider using `XCTAssert`

Reply:
Can't this be just:
```suggestion
        XCTAssertTrue(mockChatClient.mockExtensionLifecycle.receivedIsReceivingEvents)
```

🤔

Reply:
@alteral it cannot because the value is optional